config/projects.yml

```diff
@@ -259,7 +259,7 @@ media-bundle:
       php: ['5.3', '5.4', '5.5', '5.6', '7.0']
       target_php: 5.6
       versions:
-        symfony: ['2.3', '2.7', '2.8', '3.0', '3.1']
+        symfony: ['2.8', '3.0', '3.1']
         doctrine_odm: ['1']
         sonata_core: ['3']
         sonata_admin: ['3']
```

Comment:
Yeah, so… what's the point? 3.x is the default I think, and is used on almost every other build